content/docs/overview/tracers.md

```diff
@@ -1,3 +1,69 @@
 ---
 title: Tracers
 ---
+
+## Introduction
+
+OpenTracing provides an open, vendor-neutral standard API for describing distributed transactions, specifically causality, semantics and timing. It provides a general purpose distributed context propagation framework, consisting of API primitives for:
+
+- passing the metadata context in-process
+- encoding and decoding the metadata context for transmitting it over the network for inter-process communications
+- causality tracking: parent-child, forks, joins
+
+OpenTracing abstracts away the differences among numerous tracer implementations. This means that instrumentation would remain the same irrespective of the tracer system being used by the developer. In order to instrument a web application using OpenTracing specification, a compatible OpenTracing tracer must be deployed and listening for incoming span requests. A list of the all the supported tracers is available [here](./../../supported-tracers).
```

Comment:
"instrument a web application" - why web?

"and listening for incoming span requests" - doesn't seem to belong.